src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -1041,6 +1041,8 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // This Validate is needed for proper data type propagation after applying IncreasePositionIdsPrecision pass
         manager.register_pass<ov::pass::Validate>();
 
+        manager.register_pass<ov::pass::GLUFusion>();
```

Comment:
This movement will change the behavior. 
For example, we do mlp fusion 
1) At beginining gate/up layers are separate as follows 
FC_gate --- swish ---- multiply 
FC_up   ------------------/
2) And then after horizontal fusion of mlp layer it becomes 
FC_gate_and_up ->swiglu 

However if swiglu fusion is only done before the horizontal fusion, it won't be fused .

So I would like to suggest to do GLUFusion again after horizonta fusion.